CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Changelog
 
 ## vNext
 
+### Fixes
```

Comment:
@DeeJayElly please move the "Fixes" section below the "Features" 🙏

Reply:
Updated